docker/Dockerfile.contrib

```diff
@@ -11,8 +11,8 @@ RUN git clone -b ${Z2M_BRANCH} --depth 1 https://github.com/zwave-js/zwavejs2mqt
 
 ##### LOCAL SOURCE #####
 FROM node:erbium-buster AS local-copy-src
-COPY --chown=node node-zwave-js /home/node/node-zwave-js
-COPY --chown=node zwavejs2mqtt /home/node/zwavejs2mqtt
+COPY --from=git-clone-src --chown=node /home/node/node-zwave-js /home/node/node-zwave-js
```

Comment:
This will not work when using local-copy-src since it will take the cloned source